Jenkinsfile

```diff
@@ -16,6 +16,7 @@ pipeline {
         stage('sphinx') {
           agent { label 'small' }
           steps {
+            sh 'rm -rf ./blackbox/.venv'
             sh './blackbox/bin/sphinx'
```

Comment:
After removing the virtualenv, how can `./blackbox/bin/sphinx` work? Does it bootstrap itself?